### DISCUSSION TITLE
Setup branch rules

### DISCUSSION THREAD

**Discussion by Emengkeng** [fvst-dev/fvst-monorepo#98] [Ideas]
I know this might sound stupid, but please can you provide docs for setting up branch rules to either instill the Release train Feature releases stategy.

i have researched on how to implement this and have i done so(but i am not confident enough). 

also if the doc can cover how to make sure only specific users can approve merges. 

Thanks.

**Comment by peeter-tomberg** [comment]
Hey! I don't fully understand what you are asking, can you please clarify what you are looking for?
